docs/network/nodes.html

```diff
@@ -133,7 +133,8 @@
       background: 'none',
       strokeWidth: 0, // px
       strokeColor: '#ffffff',
-      align: 'horizontal'
+      align: 'horizontal',
+      textAlign: 'center'
```

Comment:
shouldn't we just use the align for this? Align is currently used by edges only. We could use it for your case when we're using nodes.